pkg/cmd/factory/remote_resolver.go

```diff
@@ -75,18 +74,18 @@ func (rr *remoteResolver) Resolver() func() (context.Remotes, error) {
 		// For enviornment default host (GH_HOST) do not fallback to cachedRemotes if none match
 		if src != "" {
 			filteredRemotes := cachedRemotes.FilterByHosts([]string{defaultHost})
-			if src == GH_HOST || len(filteredRemotes) > 0 {
+			if config.IsHostEnv(src) || len(filteredRemotes) > 0 {
 				cachedRemotes = filteredRemotes
 			}
 		}
 
 		if len(cachedRemotes) == 0 {
-			if src == GH_HOST {
-				remotesError = errors.New("none of the git remotes configured for this repository correspond to the GH_HOST environment variable. Try adding a matching remote or unsetting the variable.")
-			} else {
-				remotesError = errors.New("none of the git remotes configured for this repository point to a known GitHub host. To tell gh about a new GitHub host, please use `gh auth login`")
+			if config.IsHostEnv(src) {
+				return nil, fmt.Errorf("none of the git remotes configured for this repository correspond to the %s environment variable. Try adding a matching remote or unsetting the variable.", src)
+			} else if v, src, _ := cfg.GetWithSource("example.com", "oauth_token"); v != "" && config.IsEnterpriseEnv(src) {
```

Comment:
what's up with `example.com` here?

Reply:
hello we have to understand the server and the owner and about web browsing and about the so-called linking and capture system and their existence needs change and new enhancement to the grasp  It's fast and works without hesitation. I don't mind about the goal in the development system to share with the community

Reply:
@vilmibm Good spot! I'm using GetWithSource to check whether an enterprise token was set via the environment. Env variables like GH_ENTERPRISE_TOKEN will only get consulted if the host argument is non-blank and is a non-github.com host. Since I do not know the actual hostname in this block and since it doesn't matter, I've used an arbitrary hostname. I should probably make a code comment for it.